src/components/Buttons/ButtonPost.cy.tsx

```diff
@@ -1,19 +1,28 @@
 import React from "react";
-import ButtonPost from "./ButtonPost";
+import ButtonPost, { Props } from "./ButtonPost";
```

Comment:
I think this is already in main but can you change it to @ import?

Reply:
Refactored